README.md

```diff
@@ -220,3 +220,4 @@ curl -X POST http://localhost:11434/api/generate -d '{
 - [Typescript UI](https://github.com/ollama-interface/Ollama-Gui?tab=readme-ov-file)
```

Comment:
The project is already linked. Perhaps you want to change the name?

Reply:
I see the name has been changed, but yes it would be what it was last time `Ollama GUI Interface`

Reply:
@mchiang0610 made the change so I'll let him weigh in

Reply:
Hey thank you. We just don’t want to confuse users that this is an official project. Thank you!